handler/permissionsHandler.go

```diff
@@ -48,15 +54,33 @@ func PermittedUserFeature(attemptedFeature string, config *types.DerekRepoConfig
 }
 
 func readConfigFromURL(client http.Client, url string) []byte {
-	req, _ := http.NewRequest(http.MethodGet, url, nil)
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		log.Fatalln(err)
+	}
 
+	return fetchConfig(client, req)
+}
+
+func readConfigFromURLWithToken(client http.Client, url string, accessToken string) []byte {
+	req, err := http.NewRequest(http.MethodGet, url, nil)
```

Comment:
Is this part of the go-github SDK? I expected it to be..

Reply:
This function, or a similar one? https://github.com/google/go-github/blob/master/github/repos_contents.go#L116

It may call the same URL eventually as what you're doing manually. WDYT?

Reply:
I think you are right. We can instantiate a github client using `auth.MakeClient` and use the sdk to download the file instead of using an access token and building our own request. I'll try it out now, thanks! :+1:

Reply:
I just saw there is a `makeClient` in commentsHandler.go that builds a new client :smile:

Reply:
Updated it. What do you think?